src/Core.pm

```diff
@@ -1651,20 +1678,31 @@ sub RunCommand
   my $command = shift;
   my $try = shift;
 
+  my $lang = $self->GetSysconfigVar('language', 'RC_LANG');
```

Comment:
can it be somehow unified with https://github.com/openSUSE/perl-bootloader/blob/sle12-sp1/src/Tools.pm#L866 which do exactly same stuff?